src/main/kotlin/day12/PracJson.kt

```
@@ -0,0 +1,82 @@
+               {
+                 "price": 60.00,
+                 "cvtDatetime": "2023-03-26T08:08:45"
+               }
+             ]
+           }
+         ]
+        }
+    """
+        )
+        println(data)
+    } catch (e: Exception) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)